src/CommunityToolkit.Maui/Behaviors/Validators/MultiValidationBehavior.shared.cs

```diff
@@ -65,7 +65,7 @@ protected override async ValueTask<bool> ValidateAsync(object? value, Cancellati
 			return c.ValidateNestedAsync(token).AsTask();
 		})).ConfigureAwait(false);
 
-		var errors = children.Where(c => c.IsNotValid).Select(GetError);
+		var errors = children.Where(c => c.IsNotValid).Select(GetError).ToList();
```

Comment:
It could be a performance regression, so I would ask, do we need to make it a concrete collection here?

From the diff, It's needed when inside the `if`, at line 78. So, it did allocate the collection in a specific case.

Reply:
It's actually a performance bump recommended by reshapes since every path in the code was making multiple calls to the same IEnumerable.